instrumentation/lettuce/lettuce-5.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_0/LettuceSyncClientTest.java

```diff
@@ -128,7 +131,7 @@ void testConnectException() {
                                             "io.netty.channel.AbstractChannel.AnnotatedConnectException"),
                                         equalTo(
                                             AttributeKey.stringKey("exception.message"),
-                                            "Connection refused: "
+                                            "Connection refused: getsockopt: "
```

Comment:
this needs to allow both the old and new message (as both are possible depending on platform)